tests/AndroidXMigrationTests/Aarxersise.Java.AndroidX/app/src/main/java/com/xamarin/aarxercise/SimpleFragment.java

```diff
@@ -17,7 +18,7 @@ public Fragment getSomething() {
         return null;
     }
 
-    public Fragment getAnotherThing() {
+    public Space getAnotherThing() {
```

Comment:
Many tests were failing due to some issue with the mapping of Fragment. This resulted in more failures and hid the real error. Adding this to reduce the error count and clearly expose the actual issue.

Reply:
`Fragment`and its derived classes were missing because binderator skipped reference (for unknown reason), so ProjectReference was not added causing missing interfaces and classes, thus invalidating numerous MCW generated code.

Fixed.

Reply:
Nice. Sometimes Google does actually have a bad maven dependency tree.